### PR TITLE
Create gravitational_potential_energy.py

### PR DESCRIPTION
**Pull request by SparshRastogi**
Finding the gravitational potential energy of an object with reference to the earth,by taking its mass and height above the ground as input

### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: physics/gravitational_potential_energy.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    19600.0
+    """
+    if mass < 0:
+        raise ValueError("The mass of a body cannot be negative")
+    if height < 0:
+        raise ValueError("The height above the ground cannot be negative")
+    return mass * 9.8 * height
+
+
+if __name__ == "__main__":
```
Comment (CaedenPH) [inline physics/gravitational_potential_energy.py:54]:
```suggestion
if __name__ == "__main__":
    from doctest import testmod
```

File: physics/gravitational_potential_energy.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    >>> potential_energy(2,8)
+    156.8
+    >>> potential_energy(20,100)
+    19600.0
+    """
+    if mass < 0:
+        raise ValueError("The mass of a body cannot be negative")
+    if height < 0:
+        raise ValueError("The height above the ground cannot be negative")
+    return mass * 9.8 * height
```
Comment (CaedenPH) [inline physics/gravitational_potential_energy.py:51]: Could you be more specific than `9.8`?
For example
```py
>>> from scipy import constants
>>> constants.g
9.80665
```

File: physics/gravitational_potential_energy.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+where m is the object's mass, g=GM/R² is the gravity of Earth, and h is the height of the object's center of mass above a chosen reference level.
+
+
+Reference : "https://en.m.wikipedia.org/wiki/Gravitational_energy"
+
+
+"""
+
+
+def potential_energy(
```
Comment (CaedenPH) [inline physics/gravitational_potential_energy.py:28]: You could also make it so that it can calculate energy perhaps

**Comment by CaedenPH** [comment]
Note that a lot of your lines are too long, hence the ``pre-commit`` failing

File: physics/gravitational_potential_energy.py (hunk 3 above)
Comment (SparshRastogi) [reply physics/gravitational_potential_energy.py:23]: You mean between two masses?

File: physics/gravitational_potential_energy.py (hunk 2 above)
Comment (SparshRastogi) [reply physics/gravitational_potential_energy.py:46]: Ok,sure

**Pull request closed without merging**